src/OpenLoco/Widget.cpp

```diff
@@ -89,34 +89,35 @@ namespace OpenLoco::Ui
                 break;
 
             case WidgetType::panel:
-                Widget::drawPanel(context, window, this, widgetFlags, wndColour);
+                drawPanel(context, window, widgetFlags, wndColour);
                 break;
 
             case WidgetType::frame:
-                Widget::drawFrame(context, window, this, widgetFlags, wndColour);
+                drawFrame(context, window, widgetFlags, wndColour);
                 break;
 
             case WidgetType::wt_3:
-                Widget::draw_3(context, window, this, widgetFlags, wndColour, enabled, disabled, activated);
+                draw_3(context, window, widgetFlags, wndColour, enabled, disabled, activated);
                 break;
 
             case WidgetType::wt_4:
                 assert(false); // Unused
                 break;
 
+            case WidgetType::wt_5:
```

Comment:
This is a duplicate case, probably introduced by accident. CI errors here, but it can just be removed.